pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java

```diff
@@ -237,6 +239,26 @@ private void testCaseQueryWithStringResults(String predicate, String[] expectedV
     testTransformFunction(transformFunction, expectedValues);
   }
 
+  @Test
+  public void testCaseTransformationWithNullColumn() {
```

Comment:
This test exercises the `CaseTransformFunction`. We want to fix the `ScalarTransformFunctionWrapper` and add a test to exercise the `ScalarTransformFunctionWrapper` that wraps `ObjectFunctions::caseWhen`.

Reply:
If your PR is "Added calls for BaseTransformation init function for NULL support", suggest to revert this irrelevant test.